### ISSUE TITLE
CSS Selectors 4: :local-link pseudo class

### ISSUE THREAD

**Issue by CanadaHonk** [mozilla/standards-positions#862]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Selectors Level 4
* Specification or proposal URL (if available): https://drafts.csswg.org/selectors/#the-local-link-pseudo
* Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1654861
* Mozillians who can provide input (optional): @emilio

### Other information
No browsers implement this yet (afaik)

**Comment by emilio** [comment]
I don't think this is particularly problematic. It's a bit unfortunate to have to care about base uri changes etc but...
